src/bootloaders/systemd-class.c

```diff
@@ -246,7 +246,11 @@ bool sd_class_install_kernel(const BootManager *manager, const Kernel *kernel)
                                          "initrd %s/%s\n",
                                          get_kernel_destination_impl(manager),
                                          kernel->target.initrd_path);
+        } else {
+                /* Optional initrd no deps */
+                boot_manager_write_initrd_nodep(manager, writer);
```

Comment:
Did it fail, succeed? Do we continue? :)

Reply:
let me work on these changes and please continue